packages/css-data/src/property-parsers/background.ts

```diff
@@ -22,6 +21,28 @@ export const parseBackground = (
 ): {
   backgroundImage: LayersValue | InvalidValue;
   backgroundColor: RgbValue | undefined;
+} => {
+  const { backgroundImage, backgroundColor: backgroundColorRaw } =
+    backgroundToLonghand(background);
+
+  const backgroundColor = backgroundColorRaw
+    ? (parseCssValue("backgroundColor", backgroundColorRaw) as RgbValue)
+    : undefined;
+
+  return {
+    backgroundImage: parseBackgroundImage(backgroundImage),
+    backgroundColor:
+      backgroundColor && backgroundColor.type === "rgb"
+        ? backgroundColor
+        : undefined,
+  };
+};
+
+export const backgroundToLonghand = (
```

Comment:
It's backgorundImage to longhand, here is not the full background parsing algo

Reply:
I didn't add new logic for parsing per-se. I just split unwrapping the shorthand and parsing the longhand values

Reply:
I didn't add new logic for parsing per-se. I just split unwrapping the shorthand and parsing the longhand values